### ISSUE TITLE
cURL authentication applied regardless of request

### ISSUE THREAD

**Issue by cezmunsta** [openark/orchestrator#680]
A relatively recent change saw the following being used with cURL, regardless of the user options:
```
curl --basic --user "${basic_auth}"
```

This assumes that it is required and also that the `ORCHESTRATOR_API` does not have it set using `scheme://user:password@host:port/api`. In more recent versions of cURL, [`--disallow-username-in-url`](https://curl.haxx.se/libcurl/c/CURLOPT_DISALLOW_USERNAME_IN_URL.html) can be used to prevent this from being used.

In short, the client now spews errors even if the `ORCHESTRATOR_API` is set correctly and including authentication credentials.
